ti/report/parser/junit/gojunit/ingest.go

```diff
@@ -16,33 +16,48 @@ import (
 	ti "github.com/harness/ti-client/types"
 )
 
+const defaultRootSuiteName = "Root Suite"
+
+func getRootSuiteName(envs map[string]string) string {
+	if val, ok := envs["HARNESS_JUNIT_ROOT_SUITE_NAME"]; ok {
+		return val
+	}
+	return defaultRootSuiteName
```

Comment:
I still prefer to not have `defaultRootSuiteName`. You can merge if the team agrees.